src/Catalyst.Core.Modules.Consensus/Deltas/DeltaCache.cs

```diff
@@ -71,10 +92,7 @@ public static string GetLocalDeltaCacheKey(CandidateDeltaBroadcast candidate)
             _memoryCache.Set(GenesisHash, genesisDelta);
         }
 
-        private void EvictionCallback(object key, object value, EvictionReason reason, object state)
-        {
-            _logger.Debug("Evicted Delta {0} from cache.", key);
-        }
+        private void EvictionCallback(object key, object value, EvictionReason reason, object state) { _logger.Debug("Evicted Delta {0} from cache.", key); }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unused method parameter 'state'.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4979613)